plugin.rb

```diff
@@ -17,6 +17,14 @@
     require_relative "app/serializers/#{file}"
   end
 
+  if ENV['SENTRY_DSN'].present?
+    Sentry.init do |config|
+      config.dsn = ENV['SENTRY_DSN']
+      config.environment = ENV['SENTRY_ENV'] || 'staging'
+    end
```

Comment:
check the following, can enhance the tracing

```suggestion
      
      # get breadcrumbs from logs
      config.breadcrumbs_logger = [:active_support_logger, :http_logger]
      # Add data like request headers and IP for users, if applicable;
      # see https://docs.sentry.io/platforms/ruby/data-management/data-collected/ for more info
      config.send_default_pii = true
      
      config.traces_sample_rate = 0.1
      config.sample_rate = 0.1

    end
```